config/environments/production.rb

```diff
@@ -43,7 +43,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = Settings.force_ssl
+  config.force_ssl = Settings.rails.force_ssl
```

Comment:
see https://github.com/ministryofjustice/defence-request-service/blob/master/config/settings.yml#L6-L8 - where the force_ssl item is within the 'rails:' section